nats/aio/client.py

```diff
@@ -561,6 +563,7 @@ def sig_cb(nonce: str) -> bytes:
                             # into the pre allocated bytearray.
                             user_seed = bytearray(nkey_size)
                             f.readinto(user_seed)  # type: ignore[attr-defined]
+                user_seed = user_seed.strip()
```

Comment:
I think this is the fix?

Reply:
yes, all the seed  related with nkeys will all need this strip,  otherwise still cause invalid seed